classes/Hook.php

```diff
@@ -818,7 +818,7 @@ public static function exec(
                 $hookRegistry->collect();
             }
 
-            return ($array_return) ? [] : false;
+            return ($array_return) ? [] : null;
```

Comment:
In the documentation of this same function it says that it returns null in case of an invalid hook, in fact, it is done in all cases less in this line. [     * @return mixed|null Module's output](https://github.com/PrestaShop/PrestaShop/blob/5406d1fd55e1ce9432af9431910285e6eb563750/classes/Hook.php#L762)